packages/gatsby-theme-newrelic/src/components/Footer.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from './Icon';
+import Logo from './Logo';
+import ExternalLink from './ExternalLink';
+import { graphql, useStaticQuery, Link } from 'gatsby';
+import { css } from '@emotion/core';
+
+const Footer = ({ fileRelativePath }) => {
+  const { site } = useStaticQuery(graphql`
+    query FooterQuery {
+      site {
+        siteMetadata {
+          repository
```

Comment:
Can we also add a hook for `defaultBranch` in the `siteMetadata` (default the value to `main`)? Its something I've been meaning to fix. In the dev site, for example, when you edit a page, it uses `main` as the base branch, which is not what we want to accept contributions from (since we branch off of `develop`). Thoughts?

Reply:
I like this, I'll add it to the theme and use the default of `main`.